### ISSUE TITLE
System unavailable: Various pLinux, xLinux, and Mac nodes cannot be reached

### ISSUE THREAD

**Issue by adamfarley** [adoptium/infrastructure#3921]
Nagios has identified the following machines that cannot be contacted, or whose jenkins agent cannot be reached.

Please let me know if there's anything I can do to help resolve these.

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=build-marist-rhel79-s390x-1&service=Check+if+Jenkins+Agent+Connected)
(build-marist-rhel79-s390x-1) CRITICAL - Jenkins agent is fully disconnected

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=docker-skytap-ubuntu2004-ppc64le-1&service=PING)
(docker-skytap-ubuntu2004-ppc64le-1) PING CRITICAL - Packet loss = 100%
	
[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=dockerhost-skytap-ubuntu2004-ppc64le-1&service=PING)
(dockerhost-skytap-ubuntu2004-ppc64le-1) PING CRITICAL - Packet loss = 100%

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=test-skytap-ubuntu2404-ppc64le-1&service=Check+if+Jenkins+Agent+Connected)
(test-skytap-ubuntu2404-ppc64le-1) CRITICAL - Jenkins agent is fully disconnected

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=test-skytap-ubuntu2404-ppc64le-1&service=PING)
(test-skytap-ubuntu2404-ppc64le-1) PING CRITICAL - Packet loss = 100%

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=dockerhost-skytap-ubuntu2204-x64-1&service=PING)
(dockerhost-skytap-ubuntu2204-x64-1) PING CRITICAL - Packet loss = 100%

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=test-macincloud-macos1201-x64-1&service=Check+Jenkins+Connection)
(test-macincloud-macos1201-x64-1) CRITICAL - Jenkins agent is fully disconnected

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=test-macincloud-macos1201-x64-2&service=Check+Jenkins+Connection)
(test-macincloud-macos1201-x64-2) CRITICAL - Jenkins agent is fully disconnected

**Comment by adamfarley** [comment]
Note: please prioritise the pLinux docker build machines, as there are now none online:

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=Nagios_Server&service=Check+Label-+build%2FdockerBuild%2Flinux%2Fppc64le)
(Nagios_Server) CRITICAL - 0% machines online in 'build&&dockerBuild&&linux&&ppc64le' label

**Comment by sxa** [comment]
mac systems were in an odd state where they were online and jenkins seemed to be starting the agent process but the executor never came online. No obvious reason, so both have been rebooted. Noted in https://github.com/adoptium/infrastructure/issues/3746#issuecomment-2789325031

**Comment by Haroon-Khel** [comment]
>Note: please prioritise the pLinux docker build machines, as there are now none online:

[Link](https://nagios.adoptopenjdk.net/nagios/cgi-bin/extinfo.cgi?type=2&host=Nagios_Server&service=Check+Label-+build%2FdockerBuild%2Flinux%2Fppc64le)
(Nagios_Server) CRITICAL - 0% machines online in 'build&&dockerBuild&&linux&&ppc64le' label

I've added a build label to https://ci.adoptium.net/computer/dockerhost-osuosl-ubuntu2404-ppc64le-1

**Comment by Haroon-Khel** [comment]
Reiterating my comments in https://adoptium.slack.com/archives/C53GHCXL4/p1744187300652869
- build-marist-rhel79-s390x-1 has been decommissioned so it wont be coming back
- The skytap machines were lost during the data centre migration. https://ci.adoptium.net/computer/dockerhost-osuosl-ubuntu2404-ppc64le-1 is still up for builds and installer jobs, and 3 static containers are up and running on it for tests
- dockerhost-skytap-ubuntu2204-x64-1 also lost in the skytap migration, but it spent most of its life offline anyway so not much of a loss
